docs/docs/integrations/tools/salesforce.ipynb

```diff
@@ -247,32 +257,67 @@
     "from langchain_salesforce import SalesforceTool\n",
     "\n",
+    "# Initialize the Salesforce tool\n",
     "tool = SalesforceTool(\n",
     "    username=username, password=password, security_token=security_token, domain=domain\n",
     ")\n",
     "\n",
-    "llm = ChatOpenAI(model=\"gpt-4o-mini\")\n",
+    "# Initialize Anthropic LLM\n",
+    "llm = ChatAnthropic(model=\"claude-sonnet-4-20250514\")\n",
     "\n",
-    "prompt = PromptTemplate.from_template(\n",
-    "    \"What is the name of the contact with the id {contact_id}?\"\n",
-    ")\n",
+    "# First, let's query some contacts to get real data\n",
+    "contacts_query = {\n",
+    "    \"operation\": \"query\",\n",
+    "    \"query\": \"SELECT Id, Name, Email, Phone FROM Contact LIMIT 3\",\n",
+    "}\n",
     "\n",
-    "chain = prompt | tool.invoke | llm\n",
+    "contacts_result = tool.invoke(contacts_query)\n",
     "\n",
-    "result = chain.invoke({\"contact_id\": \"003XXXXXXXXXXXXXXX\"})"
+    "# Now let's use the LLM to analyze and summarize the contact data\n",
+    "if contacts_result and \"records\" in contacts_result:\n",
+    "    contact_data = contacts_result[\"records\"]\n",
+    "\n",
+    "    # Create a message asking the LLM to analyze the contact data\n",
+    "    analysis_prompt = f\"\"\"\n",
+    "    Please analyze the following Salesforce contact data and provide insights:\n",
+    "    \n",
+    "    Contact Data: {contact_data}\n",
+    "    \n",
+    "    Please provide:\n",
+    "    1. A summary of the contacts\n",
+    "    2. Any patterns you notice\n",
+    "    3. Suggestions for data quality improvements\n",
+    "    \"\"\"\n",
+    "\n",
+    "    message = HumanMessage(content=analysis_prompt)\n",
+    "    analysis_result = llm.invoke([message])\n",
+    "\n",
+    "    print(\"\\nLLM Analysis:\")\n",
+    "    print(analysis_result.content)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "b8467ae7",
    "metadata": {},
    "source": [
-    "## API reference\n",
-    "[langchain-salesforce README](https://github.com/colesmcintosh/langchain-salesforce/blob/main/README.md)"
+    "## API Reference\n",
+    "\n",
+    "For comprehensive documentation and API reference, see:\n",
+    "\n",
+    "- [langchain-salesforce README](https://github.com/colesmcintosh/langchain-salesforce/blob/main/README.md)\n",
+    "- [SalesforceTool API Documentation](https://python.langchain.com/docs/integrations/tools/salesforce/)\n",
+    "- [Simple Salesforce Documentation](https://simple-salesforce.readthedocs.io/en/latest/)\n",
+    "\n",
+    "## Additional Resources\n",
+    "\n",
+    "- [Salesforce SOQL Reference](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/)\n",
+    "- [Salesforce REST API Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/)\n",
+    "- [LangChain Tools Documentation](https://python.langchain.com/docs/modules/tools/)"
```

Comment:
I think it's better to remove the internal cross-reference to tools. Focusing on the things that are salesforce specific or integration specific is good for the additional resources.